### DISCUSSION TITLE
Color Swatch Column

### DISCUSSION THREAD

**Discussion by morganchorlton3** [RVxLab/filament-colorpicker#7] [Ideas]
Hi, I recently came across your package and thought that a swatch column would be useful, I cloned the repo and have added the code (I would be happy to create a PR, im just not sure how to test it properly as I have never work/ created a php/laravel package before) 

The Idea:
Just a simple column showing a visual representation of the color
<img width="1100" alt="Screenshot 2022-01-26 at 21 28 44" src="https://user-images.githubusercontent.com/32246773/151250096-a1117776-3e8e-4ba7-b525-14d6abe7a82b.png">

not sure what you think if it would be useful

**Comment by RVxLab** [comment]
I actually had the idea to add this soon so I’d gladly accept a PR for this! 

I don’t think tests are super necessary for this as I think this would be a rather simple addition. My only request is that the swatch is a rounded rectangle, I just think they look nicer.

**Reply by morganchorlton3** [reply]
Ahh brill yeah I can change that, how would I go about creating a branch for a PR?

**Reply by RVxLab** [reply]
Fork the repo and check out locally, make the changes and push them. Then create a pull request through GitHub. 

Are you new to using Git by chance?

**Reply by morganchorlton3** [reply]
Yeah I am doing Brill I will do that thanks 👍🏻

**Reply by RVxLab** [reply]
I expanded on your feature and tagged it as 1.2.0.

Thanks again for your contribution!

**Reply by morganchorlton3** [reply]
Like the copy feature great Idea
